Lib/test/test_email/test__header_value_parser.py

```diff
@@ -1700,6 +1700,14 @@ def test_get_display_name_ending_with_obsolete(self):
         self.assertEqual(display_name[3].comments, ['with trailing comment'])
         self.assertEqual(display_name.display_name, 'simple phrase.')
 
+    def test_get_display_name_for_invalid_address_field(self):
+        # bpo-32178: Test that address fields starting with `:` don't cause
```

Comment:
Since every test essentially tests something, it is redundant to mention it in a comment starting with `check if` or `test that` and the likes. I'd suggest reframing this to:

```suggestion
        # bpo-32178: Address fields starting with `:` shouldn't cause an
```

Reply:
Leading comments in test cases usually tries to answer "What does this test do?" which naturally tend to end up being "Test if" or "test that" and I disagree a few redundant extra words in the comments really matter so much. DRY principle doesn't usually apply to comments (in some cases, I'd argue reverse DRY applies to comments, but I digress :-).

I am going to skip this suggestion (and another rebuild) this time, but I'll keep this in mind next time, thanks!